po/fr.po

```diff
@@ -16,50 +16,50 @@ msgstr ""
 #: ../qml/phoneui/ui/SetPasswordPage.qml:105
 #: ../qml/phoneui/ui/SettingsPage.qml:55
 msgid "Change passphrase"
-msgstr ""
+msgstr "Modifier le mot de passe"
 
 #: ../qml/phoneui/ui/SetPasswordPage.qml:17
 #: ../qml/phoneui/ui/SettingsPage.qml:55
 msgid "Create passphrase"
-msgstr ""
+msgstr "Créer un mot de passe"
 
 #: ../qml/phoneui/ui/SetPasswordPage.qml:53
 msgid "Old passphrase"
-msgstr ""
+msgstr "Ancien mot de passe"
 
 #: ../qml/phoneui/ui/SetPasswordPage.qml:71
 #, fuzzy
 msgid "New passphrase"
-msgstr "Nouveau message"
+msgstr "Nouveau mot de passe"
 
 #: ../qml/phoneui/ui/SetPasswordPage.qml:87
 msgid "Repeat new passphrase"
-msgstr ""
+msgstr "Répétez le nouveau mot de passe"
 
 #: ../qml/phoneui/ui/SetPasswordPage.qml:105
 #: ../qml/phoneui/ui/PasswordPage.qml:82
 msgid "Submit passphrase"
-msgstr ""
+msgstr "Soumettre le mot de passe"
 
 #: ../qml/phoneui/ui/SetPasswordPage.qml:120
 #: ../qml/phoneui/ui/PasswordPage.qml:96 ../qml/phoneui/ui/PasswordPage.qml:100
 msgid "Disable passphrase?"
-msgstr ""
+msgstr "Désactiver le mot de passe ?"
```

Comment:
Thanks for translation :) is it normal in french to have a space before a ! or ? at the end of the sentence?

Reply:
Yes it is, with all double punctuations (: ; ? ! « »).
It is the contrary in English, no space!